app/models/concerns/content_types.rb

```diff
@@ -79,6 +79,21 @@ def confidence_question?
     page_type.eql?('confidence_questionnaire')
   end
 
+  # @return [Boolean]
+  def opinion_intro?
+    page_type.eql?('opinion_intro')
+  end
+
+  # @return [Boolean]
+  def one_off_question?
```

Comment:
This shouldn't be here. It only relates to questions and is already defined in `skippable?`. However, the field it queries needs updating as per another comment.